apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -209,6 +209,19 @@ ${shapeId} {
       ${sh.description} "Only published datasets will be listed in the external tools. A draft will be nevertheless be public." ;
       ${sh.group} ${mainGroup} ;
     ] ;
+
+    ${sh.property} [
+      ${sh.name} "Hide this cube?" ;
+      ${sh.path} ${cc.isHiddenCube} ;
+      ${sh.datatype} ${xsd.boolean} ;
+      ${sh.defaultValue} ${false} ;
```

Comment:
I think we discussed with @ktk that a new project should be hidden by default to prevent accidents where someone publishes data publicly.

That would apply to the shape to create projects